src/Minsk/Minsk.csproj

```
@@ -4,6 +4,19 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
+  <!--
+    HACK: Partially Fixes DesignTime Builds. See terrajobst/minsk#121
+  -->
+
+  <PropertyGroup>
+    <GeneratedSources>CodeAnalysis/**/*.g.cs</GeneratedSources>
+    <DefaultItemExcludes>$(GeneratedSources)</DefaultItemExcludes>
```

Is `DefaultItemExcludes` empty by default? Seems like we should be doing this:

```suggestion
    <DefaultItemExcludes>$(DefaultItemExcludes);$(GeneratedSources)</DefaultItemExcludes>
```

---

Don't need to prepend the property.